ocs_ci/framework/conf/default_config.yaml

```diff
@@ -16,6 +16,10 @@ RUN:
   log_dir: "/tmp"
   run_id: null  # this will be redefined in the execution
   kubeconfig_location: 'auth/kubeconfig' # relative from cluster_dir
+  # kubeadmin_password: '' # kubeadmin password used as alternative way to
```

Comment:
Update values in:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md

Reply:
Updated in [657d28f](https://github.com/red-hat-storage/ocs-ci/pull/11749/commits/657d28ff0cdeea9a3fa9394ac701699f09b92bc5).